### DISCUSSION TITLE
Gene Ontology

### DISCUSSION THREAD

**Discussion by SynatKeam** [geneontology/helpdesk#367] [Q&A] [answered]
Dear All, 

I have RNAseq data from mouse tissue and would like to perform gene ontology specifically for Biological Process. 

Basically, I performed WGCNA and identify the module that was associated with trait. Therefore, I would like to investigate whether the genes in that module is over-represented in any geneset of Gene Ontology. I performed it in ClusterProfiler and some other tool in R, but none of them worked for me. 

Now I would like to use to try using Gene Ontology interactive web-based. I am not sure how to use it yet. Here is how I started as shown in attached picture. I just copy my gene symbol and pasted in the box and clicked launch. 

![Gene Ontology](https://user-images.githubusercontent.com/86950372/143170104-435e3f9a-be75-4f7c-afb3-440dc3e07d77.JPG)

Here is the output I got. 

![GO output](https://user-images.githubusercontent.com/86950372/143170237-56596e75-f528-444c-9d71-c5d26082f43f.JPG)

Did I do it correctly? Or is there a proper way to do it? This is the only quick way I can do. I have just used it for the first time and appreciate if all member could help me.

Kind Regards,

Synat

**Comment by suzialeksander** [comment, marked as answer]
Answered by email.  In summary:

> GO works with only entrezID

I am not sure where this comes from, I suspect it's a specific tool that someone was using as it does not apply to GO in general.  I personally do not use Entrez IDs as I'm more familiar with UniProt or MOD identifiers.  The Gene Ontology is already mapped to many accessions, see http://geneontology.org/docs/faq/#how-do-i-find-the-annotations-mappings-for-entrez-ncbi-or-other-ids

As for your project, it looks like you're on the right track.  The box on our homepage actually goes to PANTHER, and we have a short info sheet available at http://geneontology.org/docs/go-enrichment-analysis/.  If you do not find what you're looking for, please see the help documents from PANTHER at www.pantherdb.org/tips/tips_geneListAnalysis.jsp as well as http://www.pantherdb.org/help/PANTHERhelp.jsp.  You can also run your EA directly on the PANTHER website at http://www.pantherdb.org/, where there are a few more ways to customise your analysis.

You might also read over http://geneontology.org/docs/ontology-documentation/, which could aid you in deciding if you'd like to look at Biological Process terms or Molecular Function terms.  It's less likely you'll be interested in the Cellular Component, but sometimes that could be useful as well.

**Comment by SynatKeam** [comment]
Thanks dear for your response.

Kind Regards,

synat
